mm2src/coins/utxo/utxo_builder/utxo_conf_builder.rs

```diff
@@ -313,4 +315,8 @@ impl<'a> UtxoConfBuilder<'a> {
     }
 
     fn avg_blocktime(&self) -> Option<u64> { self.conf["avg_blocktime"].as_u64() }
+
+    fn fee_priorities(&self) -> Option<UtxoFeePriorities> {
+        json::from_value(self.conf["fee_priorities"].clone()).unwrap_or(None)
```

Comment:
Suggestion: we could throw a error if 'fee_priorities' is parsed incorrectly (bad or omitted values)